adafruit_ahtx0.py

```diff
@@ -106,16 +106,14 @@ def reset(self) -> None:
         time.sleep(0.02)  # 20ms delay to wake up
 
     def calibrate(self) -> bool:
-        """Ask the sensor to self-calibrate. Returns True on success, False otherwise"""
+        """Ask the sensor to self-calibrate. May not 'succeed' on newer AHT20s."""
         self._buf[0] = AHTX0_CMD_CALIBRATE
         self._buf[1] = 0x08
         self._buf[2] = 0x00
         with self.i2c_device as i2c:
             i2c.write(self._buf, start=0, end=3)
```

Comment:
this may fail, so do a try/except pass here

Reply:
@ladyada ohh okay, will fix. one question though- in `__init__` if calibration returns `False` then it raises a `RuntimeError`, should that be changed too to not raise an error? or would the sensor read the SENSOR_CALIBRATED bit with the try/except and not return `False`?

Reply:
nope

![image](https://github.com/adafruit/Adafruit_CircuitPython_AHTx0/assets/1214161/30c68104-87a2-4a2b-ac21-927a2f97d5d0)